src/Swoole/SwooleClient.php

```diff
@@ -157,32 +157,36 @@ public function sendResponseHeaders(Response $response, SwooleResponse $swooleRe
      */
     protected function sendResponseContent(OctaneResponse $octaneResponse, SwooleResponse $swooleResponse): void
     {
-        if ($octaneResponse->response instanceof StreamedResponse && property_exists($octaneResponse->response, 'output')) {
-            $swooleResponse->end($octaneResponse->response->output);
-
-            return;
-        } elseif ($octaneResponse->response instanceof BinaryFileResponse) {
+        if ($octaneResponse->response instanceof BinaryFileResponse) {
             $swooleResponse->sendfile($octaneResponse->response->getFile()->getPathname());
 
             return;
         }
 
+        if ($octaneResponse->outputBuffer) {
+            $swooleResponse->write($octaneResponse->outputBuffer);
+        }
+
         if ($octaneResponse->response instanceof StreamedResponse) {
-            ob_start();
+            ob_start(function ($data) use ($swooleResponse) {
+                if ($data) {
+                    $swooleResponse->write($data);
+                }
 
-            $octaneResponse->response->sendContent();
+                return '';
+            }, 10);
```

Comment:
Is `1` safer to use? Bit edge case but if you have something like:
```php
echo '1';
sleep(5);
echo '2';
```
It doesn't return directly.

Reply:
https://user-images.githubusercontent.com/172966/114196759-3f336e00-9984-11eb-9410-0d7ac41733c7.mp4

Here's some test with `10`, not sure if `1` (byte) suitable for chunk. Maybe let's everyone try first?

Reply:
I tested with chunksize 1:

```php
    return \response()->stream(function(){

       echo str_repeat('T', random_int(1, 1000));
       sleep(2);
        echo str_repeat('E', random_int(1, 1000));
        sleep(2);
        echo str_repeat('S', random_int(1, 1000));
        sleep(2);
        echo str_repeat('T', random_int(1, 1000));
        sleep(2);
       echo 'done';
    });
```
And added some logging to the ob callback:
```php
ob_start(function($data) use($swooleResponse) {
                logger('write ' . strlen($data));
                if (strlen($data)) {
                    $swooleResponse->write($data);
                }

                return '';
            }, 1);
```

It still chunks to parts for me:
```
[2021-04-09 14:45:35] local.DEBUG: write 676  
[2021-04-09 14:45:37] local.DEBUG: write 601  
[2021-04-09 14:45:39] local.DEBUG: write 492  
[2021-04-09 14:45:41] local.DEBUG: write 4  
[2021-04-09 14:45:41] local.DEBUG: write 0  
```

So with chunkSize 10 I think it's possible that small parts are stuck untill a new item is flushed.